lib/fluent/plugin/in_tail.rb

```diff
@@ -120,26 +119,20 @@ def configure_tag
     def start
       super
 
+      @parser = parser_create(conf: @config.elements('parse').first)
+
       if @pos_file
         @pf_file = File.open(@pos_file, File::RDWR|File::CREAT|File::BINARY, @file_perm)
         @pf_file.sync = true
         @pf = PositionFile.parse(@pf_file)
       end
 
-      @loop = Coolio::Loop.new
       refresh_watchers
-
-      @refresh_trigger = TailWatcher::TimerWatcher.new(@refresh_interval, true, log, &method(:refresh_watchers))
-      @refresh_trigger.attach(@loop)
-      @thread = Thread.new(&method(:run))
+      timer_execute(:refresh_watchers, @refresh_interval, &method(:refresh_watchers))
```

Comment:
Time title is global. So title with `tail_` prefix is better than one without it.